GTG/gtk/editor/editor.py

```diff
@@ -333,40 +403,40 @@ def refresh_editor(self, title=None, refreshtext=False):
         # as done, we display the number of days left.
-        if status in [Task.STA_DISMISSED, Task.STA_DONE]:
-            delay = self.task.get_days_late()
-            if delay is None:
-                txt = ""
-            elif delay == 0:
-                txt = "Completed on time"
-            elif delay >= 1:
-                txt = ngettext("Completed %(days)d day late",
-                               "Completed %(days)d days late", delay) % \
-                    {'days': delay}
-            elif delay <= -1:
-                abs_delay = abs(delay)
-                txt = ngettext("Completed %(days)d day early",
-                               "Completed %(days)d days early", abs_delay) % \
-                    {'days': abs_delay}
-        else:
-            due_date = self.task.get_due_date()
-            result = due_date.days_left()
-            if due_date.is_fuzzy():
-                txt = ""
-            elif result > 0:
-                txt = ngettext("Due tomorrow!", "%(days)d days left", result) \
-                    % {'days': result}
-            elif result == 0:
-                txt = _("Due today!")
-            elif result < 0:
-                abs_result = abs(result)
-                txt = ngettext("Due yesterday!", "Was %(days)d days ago",
-                               abs_result) % {'days': abs_result}
-
-        style_context = self.window.get_style_context()
-        color = style_context.get_color(Gtk.StateFlags.INSENSITIVE).to_color()
-        self.dayleft_label.set_markup(
-            "<span color='%s'>%s</span>" % (color.to_string(), txt))
+        # if status in [Task.STA_DISMISSED, Task.STA_DONE]:
+        #     delay = self.task.get_days_late()
+        #     if delay is None:
+        #         txt = ""
+        #     elif delay == 0:
+        #         txt = "Completed on time"
+        #     elif delay >= 1:
+        #         txt = ngettext("Completed %(days)d day late",
+        #                        "Completed %(days)d days late", delay) % \
+        #             {'days': delay}
+        #     elif delay <= -1:
+        #         abs_delay = abs(delay)
+        #         txt = ngettext("Completed %(days)d day early",
+        #                       "Completed %(days)d days early", abs_delay) % \
+        #             {'days': abs_delay}
+        # else:
+        #     due_date = self.task.get_due_date()
+        #     result = due_date.days_left()
+        #     if due_date.is_fuzzy():
+        #         txt = ""
+        #     elif result > 0:
+        #         txt=ngettext("Due tomorrow!", "%(days)d days left", result) \
+        #             % {'days': result}
+        #     elif result == 0:
+        #         txt = _("Due today!")
+        #     elif result < 0:
+        #         abs_result = abs(result)
+        #         txt = ngettext("Due yesterday!", "Was %(days)d days ago",
+        #                        abs_result) % {'days': abs_result}
```

Comment:
Can you refactor the computation of this label in a separate function? It should have been there in the first place. You don't have to call that function, just put a commented out call in here.